seapath_ros_driver/include/seapath_ros_driver.hpp

```diff
@@ -3,196 +3,215 @@
 
 #include <iostream>
 #include <sstream>
-#include <cmath> 
+#include <cmath>
 #include <vector>
 #include <chrono>
 #include <memory>
 
-#include <rclcpp/rclcpp.hpp> 
+#include <rclcpp/rclcpp.hpp>
 #include <sensor_msgs/msg/nav_sat_fix.hpp>
 #include <geometry_msgs/msg/twist_with_covariance_stamped.hpp>
 #include <geometry_msgs/msg/pose_with_covariance_stamped.hpp>
+#include <nav_msgs/msg/odometry.hpp>
 #include <tf2/transform_datatypes.h>
 #include <tf2/LinearMath/Quaternion.h>
 #include <diagnostic_msgs/msg/diagnostic_status.hpp>
 #include <diagnostic_msgs/msg/diagnostic_array.hpp>
 #include <vortex_msgs/msg/km_binary.hpp>
-
+#include "tf2/LinearMath/Quaternion.h"
+#include "tf2_ros/transform_broadcaster.h"
+#include "geometry_msgs/msg/transform_stamped.hpp"
 #include "seapath_socket.hpp"
```

Comment:
Very nitpicky, but try to structure this a bit more. Ome block for tf2, one for geometry_msgs etc..